File: Prefix_Sum/Pattern2/Pattern2.py
nums=[10,22,45,2,33,46,32,33,55,21,345,98]
prefix_sums=[0]*len(nums)
sum=0
start=2
end=6
for i in range(end+1):
    sum+=nums[i]
    if i>=start:
        print(sum,end=",")

File: Prefix_Sum/Pattern2/Pattern2.java
public class Pattern2 {
    public static void main(String[] args) {
        int[] nums={10,22,45,2,33,46,32,33,25,22,67,69};
        int start=3;
        int end=6;
        int sum=0;
        for(int i=0;i<end+1;i++){
            sum+=nums[i];
            if(i>=start){
                System.out.print(sum+",");
            }
        }
    }
    
}
